examples/dropwizard/.gitignore

```diff
@@ -0,0 +1,8 @@
+src/main/jib/config.yml
+
+target/
+dependency-reduced-pom.xml
```

Comment:
What is this file? `dependency-reduced-pom.xml`

Reply:
It's created by the maven-shade-plugin.